test/e2e/placement_eviction_test.go

```diff
@@ -156,14 +378,118 @@ var _ = Describe("ClusterResourcePlacement eviction of bound binding, no taint s
 		crpEvictionStatusUpdatedActual := testutilseviction.StatusUpdatedActual(
 			ctx, hubClient, crpEvictionName,
 			&testutilseviction.IsValidEviction{IsValid: true, Msg: condition.EvictionValidMessage},
-			&testutilseviction.IsExecutedEviction{IsExecuted: true, Msg: condition.EvictionAllowedNoPDBMessage})
+			&testutilseviction.IsExecutedEviction{IsExecuted: false, Msg: fmt.Sprintf(condition.EvictionBlockedPDBSpecifiedMessageFmt, 3, 3)})
 		Eventually(crpEvictionStatusUpdatedActual, eventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to update cluster resource placement eviction status as expected")
 	})
 
-	It("should ensure evicted cluster is picked again by scheduler & update cluster resource placement status as expected", func() {
+	It("should ensure cluster resource placement status is unchanged", func() {
+		crpStatusUpdatedActual := crpStatusUpdatedActual(workResourceIdentifiers(), allMemberClusterNames, nil, "0")
+		Eventually(crpStatusUpdatedActual, eventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to update cluster resource placement status as expected")
+	})
+
+	It("should still place resources on the all available member clusters", checkIfPlacedWorkResourcesOnAllMemberClusters)
+})
+
+var _ = Describe("ClusterResourcePlacement eviction of bound binding - PickN CRP, PDB with MaxUnavailable specified as percentage, eviction allowed", Ordered, Serial, func() {
```

Comment:
add a test case of "PickN CRP, PDB with MaxUnavailable specified as percentage, eviction blocked"?